Automattic/wp-calypso
# Plugin Card: Update activa/inactive tag layout

#### Proposed Changes

* Update Installed icon and active/inactive tag layout
* Hide plugin ratings when there is active/inactive information to be shown

| Before  | After `active` | After `inactive`| After multi-site view|
| ------------- | ------------- |------------- |------------- |
|<img width="514" alt="CleanShot 2023-01-23 at 11 07 20@2x" src="https://user-images.githubusercontent.com/5039531/214074162-f5c71645-26ce-45a1-a024-433bfdc47588.png">|<img width="517" alt="CleanShot 2023-01-24 at 11 34 34@2x" src="https://user-images.githubusercontent.com/5039531/214337436-672ef86c-49ea-4a78-9b6f-b9e38519e9b5.png">|<img width="515" alt="CleanShot 2023-01-24 at 11 32 22@2x" src="https://user-images.githubusercontent.com/5039531/214337118-94573fe5-6e7f-4235-a1f4-b158b310dda2.png">|<img width="519" alt="CleanShot 2023-01-24 at 11 32 42@2x" src="https://user-images.githubusercontent.com/5039531/214337157-f115ad6e-5a70-4ecf-8416-f824be9a31a9.png">|

#### Testing Instructions
* Install a plugin
* Search for its name on `/plugins`
* Check if the active version is similar to above (without ratings)
* Enter on the plugin page and deactivate it
* Go back to the search page and check if the inactive version is similar to above (without ratings)
* Select the multi-site view on the sidebar
* Check if the multi-site view is similar to above (with ratings)

#### Pre-merge Checklist





Fixes #72340
